app/main.py

```diff
@@ -1,3 +1,12 @@
 def count_occurrences(phrase: str, letter: str) -> int:
```

Comment:
The function `count_occurrences` is defined but not implemented. You need to add logic to count the occurrences of the letter in the phrase, ensuring the operation is case-insensitive. Consider using Python's string methods like `str.lower()` or `str.upper()` to handle case insensitivity.